ui/src/features/common/commit-info.tsx

```diff
@@ -1,6 +1,7 @@
 import { GitCommit } from '@ui/gen/v1alpha1/generated_pb';
+import { PlainMessage } from '@ui/utils/connectrpc-extension';
 
-export const CommitInfo = ({ commit }: { commit: GitCommit }) => (
+export const CommitInfo = ({ commit }: { commit: PlainMessage<GitCommit> }) => (
```

Comment:
I understand the purpose of having PlainMessage, but in some places, it seems redundant.
This is an example.